src/main/scala/chisel3/iotesters/VerilatorBackend.scala

```diff
@@ -110,7 +110,7 @@ class GenVerilatorCppHarness(
     val dutName = dut.name
     val dutApiClassName = dutName + "_api_t"
     val dutVerilatorClassName = "V" + dutName
-    val widthMap = (circuit.modules flatMap findWidths).toMap
+    // val widthMap = (circuit.modules flatMap findWidths).toMap
```

Comment:
Just curious: why are the lines commented out instead of deleted?

The code is in the git history if we need it and commented out code, makes the master branch less readable.